### DISCUSSION TITLE
Upgrade Alpine to 3.15

### DISCUSSION THREAD

**Discussion by mikekuzak** [haugene/docker-transmission-openvpn#2134] [General]
The current image has a lot of CVEs

```
haugene/transmission-openvpn:3.7.1 (alpine 3.13.6)
==================================================
Total: 28 (HIGH: 26, CRITICAL: 2)
```

**Comment by edgd1er** [comment]
alpine is only used at build time. ubuntu is the distribution used at runtime: 

https://github.com/haugene/docker-transmission-openvpn/blob/20877f1b168b6ff27fc58aeef40756572c562d47/Dockerfile#L19

**Reply by pkishino** [reply]
This, we have deprecated alpine (was used pre-4.0 release) and those are no longer maintained

**Comment by mikekuzak** [comment]
So with the move to ubuntu Transmission has been downgraded from 3.0.0 to 2.9.4 Any reason for this move ?

**Reply by pkishino** [reply]
Yes, please see the main 4.0 release thread

**Comment by mikekuzak** [comment]
This looks ok, thanks
```
haugene/transmission-openvpn:4.0 (ubuntu 20.04)
===============================================
Total: 0 (HIGH: 0, CRITICAL: 0)

ghcr.io/linuxserver/swag:latest
2021-12-30T12:09:23.524Z	[34mINFO[0m	Number of language-specific files: 1
2021-12-30T12:09:23.525Z	[34mINFO[0m	Detecting python-pkg vulnerabilities...
```
